docs/json/radarr/cf/upscaled.json

```diff
@@ -51,6 +51,24 @@
       "fields": {
         "value": "(?<=\\b[12]\\d{3}\\b).*\\b(UPS|Up(s(caled?|UHD)|(Rez)))\\b"
       }
+    },
+    {
+      "name": "Ai-Enhanced",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": false,
+      "fields": {
+        "value": "\\bAi-Enhanced\\b"
+      }
+    },
+    {
+      "name": "DotAi",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": false,
+      "fields": {
+        "value": "\\.Ai\\."
```

Comment:
This is too generic and will trigger on any episode or movie title with AI in the name

Reply:
that's a good point, did not consider that. Will update when I get the chance or if someone would like to directly commit to the PR that would be great too